src/Raven.Server/Documents/Indexes/Configuration/SingleIndexConfiguration.cs

```diff
@@ -20,7 +20,23 @@ public SingleIndexConfiguration(IndexConfiguration clientConfiguration, RavenCon
         {
             _databaseConfiguration = databaseConfiguration;
 
-            Initialize(key => clientConfiguration.GetValue(key) ?? databaseConfiguration.GetSetting(key), throwIfThereIsNoSetMethod: false);
+            Initialize(key =>
+            {
+                var clientValue = clientConfiguration.GetValue(key);
+                if (clientValue != null)
+                    return clientValue;
+
+                if (key == Constants.Configuration.Indexing.MaxMapIndexOutputsPerDocument ||
+                    key == Constants.Configuration.Indexing.MaxMapReduceIndexOutputsPerDocument)
+                {
+                    clientValue = clientConfiguration.GetValue(Constants.Configuration.Indexing.MaxIndexOutputsPerDocument);
```

Comment:
Why is this merging both `MaxIndexOutputsPerDocument` and `MaxMapReduceIndexOutputsPerDocument` to a single value?